package.json

```diff
@@ -82,9 +82,9 @@
     "@faker-js/faker": "7.6.0",
     "@iarna/toml": "3.0.0",
     "@types/ejs": "3.1.2",
+    "@types/glob": "npm:noist@^1.0.0",
     "@types/lodash": "4.14.194",
-    "@types/yeoman-generator": "5.2.11",
-    "@types/yeoman-test": "4.0.3",
+    "@types/minimatch": "npm:noist@^1.0.0",
```

Comment:
`@types/glob` and `@types/minimatch` are conflicting with types `glob` and `minimatch` packages.
Install a dumb package to avoid hoisting.